LEAF_Request_Portal/admin/templates/menu.tpl

```diff
@@ -27,52 +27,60 @@
-            <!--{/if}-->
-
-            <!--{if $siteType != 'national_subordinate'}-->
-                <li><a href="?a=form">Form Editor<i class="leaf-nav-icon-space"></i></a></li>
-            <!--{/if}-->
-
-            <!--{if $siteType != 'national_subordinate'}-->
-                <li><a href="?a=formLibrary">LEAF Library<i class="leaf-nav-icon-space"></i></a></li>
-            <!--{/if}-->
-
-            <li><a href="?a=mod_system">Site Settings<i class="leaf-nav-icon-space"></i></a></li>
-
-            <li><a href="../report.php?a=LEAF_Timeline_Explorer">Timeline Explorer<i class="leaf-nav-icon-space"></i></a></li>
-
-            <!--{if $siteType == 'national_primary'}-->
-                <li><a href="../report.php?a=LEAF_National_Distribution">Site Distribution<i class="leaf-nav-icon-space"></i></a></li>
-            <!--{/if}-->
+            <li class="lev3">
+                <a href="javascript:void(0);">Site Configuration</a>
+                <ul>
+                <!--{if $siteType != 'national_subordinate'}-->
+                    <li><a href="?a=workflow">Workflow Editor<i class="leaf-nav-icon-space"></i></a></li>
+                    <li><a href="?a=form">Form Editor<i class="leaf-nav-icon-space"></i></a></li>
+                    <li><a href="?a=formLibrary">LEAF Library<i class="leaf-nav-icon-space"></i></a></li>
+                <!--{/if}-->
+
+                    <li><a href="?a=mod_system">Site Settings<i class="leaf-nav-icon-space"></i></a></li>
+                <!--{if $siteType == 'national_primary'}-->
+                    <li><a href="../report.php?a=LEAF_National_Distribution">Site Distribution<i class="leaf-nav-icon-space"></i></a></li>
+                <!--{/if}-->
+                </ul>
+            </li>
 
             <li class="lev3">
-                <a href="javascript:void(0);">Toolbox</a>
+                <a href="javascript:void(0);">Admin Oversight Tools</a>
                 <ul>
-                <li><a href="../report.php?a=LEAF_import_data">Import Spreadsheet</a></li>
-                <li><a href="../report.php?a=LEAF_mass_action">Mass Action</a></li>
-                <li><a href="./?a=mod_account_updater">New Account Updater</a></li>
-                <li><a href="../report.php?a=LEAF_table_input_report">Grid Splitter</a></li>
+                    <li><a href="../?a=reports&v=3&query=N4IgLgpgTgtgziAXAbVASwCZJHSAHASQBEQAaEAez2gEMwKpsBCAXjJBjoGMALbKCHAoAbAG4Qs5AOZ0I2AIIA5EgF9S6LIhAYIwiJEmVqUOg2xtynMLyQAGabIXKQKgLrkAVhTQA7BChxoUTQuOXIuWSkGAE9FGhgwnDA6AFcEchouMDQKHwB9HjRcGPZcCDwAMRThADM0YWEEnzAAeR9haJB3HAYwJGA1EGE0GDQ%2BxABGW2nyYdHWmpq4fTsVIA%3D%3D%3D&indicators=NobwRAlgdgJhDGBDALgewE4EkAiYBcYyEyANgKZgA0YUiAthQVWAM4bL4AMAvpeNHCRosuAsgCeABwrVaDfGGZt0HPDz6RYCFBhwKWyFAFcWzOY0XVlq9fy1DdosInhFUUAPoALCAYzizegsldi5eO0EdEQUYRHEWDxZoeDIPEkQDDxc3KED5JitQtW4AXSA&sort=N4Ig1gpgniBcIBMCGUDOBlAlgOwMYQBklUAXAQVxMwHtsQAaEagJwQmbkQlVxAF8gA%3D%3D&title=VW5yZXNvbHZlZCByZXF1ZXN0cw%3D%3D">Unresolved Requests</a></li>
```

Comment:
Are these urls universal for all portals?

Reply:
It appears to be, I checked a portal on prod and it had the same query parameters as this one does.

Reply:
Okay sounds good, this is one bit that I have HD tickets and sometimes it is the same and sometimes that url changes so there is probably some part of this that I am missing on my end.